### PR TITLE
Section 5.7

### LINKED ISSUE #1530: Add 5.8. Content Resource section?
To discuss referenced content resources for consistency with Section 2 ?

### PR DESCRIPTION
**Pull request by azaroth42**
Add Section 5.7 and warning for `label` :( 

Closes #1530

### CODE REVIEW

File: source/api/presentation/3.0/index.md [hunk 1]
```diff
@@ -1347,22 +1351,47 @@ An Annotation Page _MUST_ have an HTTP(S) URI given in `id`, and _MAY_ have any
...
 
+The type of the content resource _MUST_ be included, and _SHOULD_ be taken from the table listed under the definition of `type`. The `format` of the resource _SHOULD_ be included and, if so, _SHOULD_ be the media type that is returned when the resource is dereferenced. The `profile` of the resource, if it has one, _SHOULD_ also be included. Resources in appropriate formats _MAY_ also have the `language`, `height`, `width`, and `duration` properties. Content resources _MAY_ also have descriptive and linking properties, as defined in [section 3][prezi-api-3-resource-properties].
+
+If the content resource is an Image, and a IIIF Image service is available for it, then the URI _MAY_ be a complete URI to any particular representation made available, such as `https://example.org/image1/full/1000,/0/default.jpg`, but _MUST NOT_ be just the URI of the IIIF Image service. It _MUST_ have a `type` of "Image". Its media type _MAY_ be listed in `format`, and its height and width _MAY_ be given as integer values for `height` and `width` respectively. The image _SHOULD_ have the service referenced from it using the `service` property.
```
Comment (zimeon) [inline source/api/presentation/3.0/index.md]: Old school "Image" --> `Image`

File: source/api/presentation/3.0/index.md [hunk 2]
```diff
@@ -1347,22 +1351,47 @@ An Annotation Page _MUST_ have an HTTP(S) URI given in `id`, and _MAY_ have any
...
+Content resources _MUST_ have an `id` property, with the value being the URI at which it can be obtained. A Canvas _MAY_ be treated as a content resource for the purposes of annotating it on to other Canvases. In this situation, the Canvas _MAY_ be embedded within the Annotation, or require dereferencing to obtain its description.
 
+The type of the content resource _MUST_ be included, and _SHOULD_ be taken from the table listed under the definition of `type`. The `format` of the resource _SHOULD_ be included and, if so, _SHOULD_ be the media type that is returned when the resource is dereferenced. The `profile` of the resource, if it has one, _SHOULD_ also be included. Resources in appropriate formats _MAY_ also have the `language`, `height`, `width`, and `duration` properties. Content resources _MAY_ also have descriptive and linking properties, as defined in [section 3][prezi-api-3-resource-properties].
```
Comment (zimeon) [inline source/api/presentation/3.0/index.md]: One sentence starts "Resources..", suggest being consistent with "Content resources..."

File: source/api/presentation/3.0/index.md [hunk 3]
```diff
@@ -1321,6 +1321,10 @@ Clients _SHOULD_ process the Annotation Pages and their items in the order given
 
 An Annotation Page _MUST_ have an HTTP(S) URI given in `id`, and _MAY_ have any of the other properties defined in this specification or the Web Annotation specification. The Annotations are listed in the `items` property of the Annotation Page.
 
+__Incompatibility Warning__
+In part due to the timing of the release of the JSON-LD and Web Annotation specifications, the definition of `label` in the Web Annotation specification does not produce JSON conformant with the structure defined in this specification. Given the absolute requirement for internationalized labels and the strong desire for consistently handing properties, the `label` property on Annotation Pages, Annotations, content resources referenced from Annotations, and Annotation Collections does not conform to the string requirement of the Web Annotation Data Model.  This [issue has been filed with the W3C][webanno-label-issue] and will hopefully be addressed in a future version of the standard.
```
Comment (mikeapp) [inline source/api/presentation/3.0/index.md]: handing -> handling

File: source/api/presentation/3.0/index.md (hunk 1 above)
Comment (tomcrane) [reply source/api/presentation/3.0/index.md]: Just "Image" --> Image, not `Image`

File: source/api/presentation/3.0/index.md [hunk 4]
```diff
@@ -1347,22 +1351,47 @@ An Annotation Page _MUST_ have an HTTP(S) URI given in `id`, and _MAY_ have any
...
 
 Note that the Web Annotation data model defines different patterns for the `value` property, when used within an Annotation. The value of a Textual Body or a Fragment Selector, for example, are strings rather than JSON objects with languages and values. Care must be taken to use the correct string form in these cases.
 
-Annotations _MUST_ have their own HTTP(S) URIs, conveyed in the `id` property. The JSON-LD description of the Annotation _SHOULD_ be returned if the URI is dereferenced, according to the [Web Annotation Protocol][webannoprotocol].
+Additional features of the [Web Annotation][webanno] data model _MAY_ also be used, such as selecting a segment of the Canvas or content resource, or embedding the comment or transcription within the Annotation. The use of these advanced features sometimes results in situations where the `target` is not a content resource, but instead a `SpecificResource`, a `Choice`, or other non-content object. Implementations should check the `type` of the resource and not assume that it is always content to be rendered.
```
Comment (tomcrane) [inline source/api/presentation/3.0/index.md]: Just SpecificResource, Choice

File: source/api/presentation/3.0/index.md (hunk 1 above)
Comment (tomcrane) [reply source/api/presentation/3.0/index.md]: >  Its `type` value _MUST_ be the string `Image`. 

Usage from backticks

**Pull request merged**